Project-Descriptions-and-Plans/CVE20/CVE20-PA-2024.md

```diff
@@ -31,14 +31,16 @@ Completing verification, including creation of a Spike reference model, and docu
 - No longer included in scope as of 2024:
  - E: Base Integer Instruction Set (embedded) with 16, 32-bit general-purpose registers
  - User Mode verification
- - CV32E40P-like interrupt interface (Open Titan CLIC)
+
  
 
 ### Component 1b "RTL design of the core complex". As of 2024 this Component is removed from the project
 
 - Removed from project scope as of 2024:
```

Comment:
The sub-bullets here do not render as expected.  I am not sure these are supported with GitHub's version of Markdown.

Reply:
@MikeOpenHWGroup Thanks for catching that. I found that if you use 2 spaces to indent the "-" instead of 1 space, it renders. Corrected.